Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -35250,4 +35250,98 @@
+        <Aliases value="_MissileHybrids"/>
+        <UnitIcon value="AP\Assets\Custom\Textures\btn-unit-hunterling.dds"/>
+        <GroupIcon>
+            <Image value="Assets\Textures\wireframe-mod-l2d_zerg-hunterling.dds"/>
+        </GroupIcon>
+        <Wireframe>
+            <Image value="Assets\Textures\wireframe-mod-l2d_zerg-hunterling.dds"/>
+        </Wireframe>
+        <LifeArmorIcon value="Assets\Textures\btn-upgrade-zerg-groundcarapace-level0.dds"/>
+        <DeathCustoms ModelLink="AP_HunterlingAirDeath" Name="Flying"/>
+        <On Terms="Behavior.AP_MercGroundDrop.Create" Send="Create AP_MercUltraliskDropModel"/>
+        <On Terms="UnitDeathCustomize; AnimPlaying Jump" Send="DeathCustomize Flying"/>
+        <On Terms="UnitDeathCustomize; IsStatus InAir 1" Send="DeathCustomize Flying"/>
+        <!--On Terms="ActorCreation" Send="TimerSet 5.000000 Ambience"/-->
+        <!--On Terms="TimerExpired; TimerName Ambience; PassChance 50.000000" Send="Create AP_Left2Die_Hunterling_Ambience"/-->
+        <!--On Terms="TimerExpired; TimerName Ambience" Send="TimerSet 5.000000 Ambience"/-->
+        <On Terms="UnitDeathCustomize; ValidateUnit AP_HunterlingLeaping" Send="DeathCustomize Flying"/>
+        <On Terms="Behavior.AP_HunterlingJump.Launch" Send="AnimBracketStart Jump Jump,Start Jump Jump,End,A"/>
+        <On Terms="Behavior.AP_HunterlingJump.Launch" Send="AnimBlendTimeApply 0.000000"/>
+        <On Terms="Behavior.AP_HunterlingJump.LandDown" Send="AnimBracketStop Jump"/>
+        <On Terms="Behavior.AP_HunterlingJump.LandDown" Send="AnimBlendTimeApply"/>
+        <On Terms="Behavior.AP_HunterlingJump.LandUp" Send="AnimBracketStop Jump"/>
+        <On Terms="Behavior.AP_HunterlingJump.LandUp" Send="AnimBlendTimeApply"/>
+        <On Terms="WeaponStart.*.AttackStart" Send="AnimBracketStart Attack Attack"/>
+        <On Terms="WeaponStop.*.AttackStop" Send="AnimBracketStop Attack"/>
+        <On Terms="Abil.AP_LeapAttack.SourceCastStart" Send="AnimClear Jump"/>
+        <On Terms="Abil.AP_LeapAttack.SourceCastStart" Send="AnimBracketStart LeapAttack Jump,Start Jump IGNORE"/>
+        <On Terms="Effect.AP_HunterlingLeapImpactDamageSearch.Start; At Caster" Send="AnimBracketStop LeapAttack"/>
+        <On Terms="Effect.AP_LeapHunterlingDamage.Start; At Caster" Send="AnimPlay PounceAttacking Spell"/>
+        <On Terms="UnitHeightUpdate.*.Land" Send="Create AP_Hunterling_Land"/>
+        <!--On Terms="Effect.LeapNoTargetDummy.Start; At Caster" Send="AnimPlay Land Jump,End,A"/>
+        <On Terms="Effect.LeapAttackDelayPersistent.Start; At Caster" Send="AnimPlay Falling Jump,End 0 0.000000 0.000000"/>
+        <On Terms="Effect.LeapAttackingPersistent.Start; At Caster" Send="AnimPlay PounceAttacking Spell PlayForever"/>
+        <On Terms="Behavior.LeapAttacking.Off" Send="AnimClear PounceAttacking"/-->
+        <Macros value="AntiGravDeathRemoveMacro"/>

```

Comment:
Add unit response sounds

Reply:
sorry, I forgot to sync one of my commits to github, which fixed sounds and some other things